docs/modules/sql/pages/get-started-sql-files.adoc

```diff
@@ -209,59 +187,122 @@ Normally, querying in SQL is database or dataset-specific. However, Hazelcast, y
 +
 [source,sql]
 ----
-CREATE MAPPING dislikes (
-name VARCHAR,
-dislikes INT
-) TYPE IMap OPTIONS ('keyFormat'='int', 'valueFormat'='json-flat');
+CREATE MAPPING names
+TYPE IMap OPTIONS ('keyFormat'='int', 'valueFormat'='varchar');
 ----
 +
-This table is mapped to a distributed map in Hazelcast where the key is an integer and the value is an object that's serialized to JSON.
+This table is mapped to a distributed map in Hazelcast where the key is an integer and the value is a string. 
 
 . Use `SINK INTO` statements to add some entries to the map.
 +
 [source,sql]
 ----
-SINK INTO dislikes VALUES
-(1, 'Greg', 1),
-(2, 'Jerry', 0),
-(3, 'Mary', 5),
-(4, 'Jerry', 0);
+SINK INTO names VALUES
+(1, 'Greg'),
+(2, 'Jerry'),
+(3, 'Mary'),
+(4, 'Jerry'),
+(5, 'Joe');
 ----
 
-. Use the xref:sql:select.adoc#join-tables[`JOIN` clause] to merge results from the `likes` and `dislikes` tables so you can see who has the most likes and dislikes.
+. Use the xref:sql:select.adoc#join-tables[`JOIN` clause] to merge results from the messages and names tables so you can see who has the most likes and dislikes.
 +
 NOTE: The data source on the right of the join must always be a map.
+
+NOTE: Because the value in the map is a single data field, it is referenced with 'this' instead of a name.
```

Comment:
I don't quite understand this sentence. Do you mean because it's a primitive data type? Either way, I don't think this note is necessary. We teach users about SQL over maps in a different tutorial.
```suggestion
```